### ISSUE TITLE
Lighting not working

### ISSUE THREAD

**Issue by adamgerhant** [gabotechs/react-stl-viewer#35]
The stl viewer was working perfectly for me for some time, but recently the displayed models are much darker and have no surface features, just a flat color.

Here's a link to what it looks like: https://drive.google.com/file/d/1uxt93zZnK8UzVrA93_mPZiHieAEbMYwn/view?usp=sharing

The code I am using: 
```
import { StlViewer } from 'react-stl-viewer';

const url = "https://storage.googleapis.com/ucloud-v3/ccab50f18fb14c91ccca300a.stl"

const style = {
    top: 0,
    left: 0,
    width: '100vw',
    height: '100vh',
}

function App() {
    return (
        <StlViewer
            style={style}
            orbitControls
            shadows
            url={url}
        />
    );
}

export default App;
```

**Comment by adamgerhant** [comment]
I also tried this on another computer and the model still has no surface.

**Comment by gabotechs** [comment]
This package has not been updated in a while, so it is very likely some compatibility issue with newer versions of three.js or maybe React.

Can you provide what versions of the relevant peerDependencies are you using?

**Comment by gabotechs** [comment]
In fact, that is the issue, I was able to replicate your problem using version `0.155.0` of `three`. If I lock the version of `three` to `0.154.0` it works correctly.

It seems like an issue with three.js, I would suggest that you lock the version in your `package.json` to:
```json
{
  ...
  "dependencies": {
    ...
    "three": "0.154.0",
    ...
  }
}
```

**Comment by gabotechs** [comment]
This could be related https://github.com/pmndrs/react-three-fiber/issues/2963

**Comment by gabotechs** [comment]
Fixed in 2.3.2
